source/_patterns/01-elements/link/link.scss

```diff
@@ -1,28 +1,34 @@
 @use "@db-ui/base/build/scss/variables" as *;
+@use "@db-ui/base/build/scss/scaling-placeholder" as *;
+@use "@db-ui/base/build/scss/color-placeholder" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
 @use "@db-ui/base/build/scss/icon/icons.helpers" as *;
 
 @import "../../../css/partials.meta";
-@import "link.variables";
 
 .elm-link {
-	border-radius: to-rem($pxValue: 4);
+	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
 	display: inline-block;
+	border-radius: to-rem($pxValue: 6);
 
 	// Link underline
-	text-underline-offset: to-rem($pxValue: 3);
+	text-underline-offset: to-rem($pxValue: 1);
 	text-underline-position: under;
+	text-decoration-thickness: to-rem($pxValue: 1);
```

Comment:
@annam002 could you please recheck those as well ?

Reply:
removed those lines in 6912f57271c0b6ed3e49daa5d1499b48e08dda69